### PR TITLE
Add tests for getController and lockController

### PR DESCRIPTION
**Pull request by joewagner**
These changes add tests that match https://github.com/tablelandnetwork/evm-tableland/pull/72 and https://github.com/tablelandnetwork/js-tableland/pull/81

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -248,7 +248,57 @@ describe('Validator, Chain, and SDK work end to end', function () {
...
+
+        const tableland = await getTableland(signer);
+
+        const prefix = 'test_create_getcontroller';
+        // `key` is a reserved word in sqlite
+        const { name } = await tableland.create('keyy TEXT, val TEXT', { prefix });
+
+        const chainId = 31337;
+        // Hardhat #7
+        const controllerAddress = '0x14dC79964da2C08b23698B3D3cc7Ca32193d9955'
```
Comment (carsonfarmer) [inline test/test.js:267]: Why hardcoded versus just pulling from the accounts list?

File: test/test.js [hunk 2]
```diff
@@ -248,7 +248,57 @@ describe('Validator, Chain, and SDK work end to end', function () {
         const { hash } = await tableland.setController('0x14dC79964da2C08b23698B3D3cc7Ca32193d9955', name);
 
         expect(typeof hash).toEqual('string');
-        expect(hash.length).toEqual(56);
+        expect(hash.length).toEqual(66);
+    });
+
+    test('get controller returns an address', async function () {
+        const wallet = new Wallet('0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d' /* Hardhat #1 */);
```
Comment (carsonfarmer) [inline test/test.js:255]: Same up here.

**Review by carsonfarmer — APPROVED** [review]
Question or two for you re: hard-coded addresses. Otherwise, LGTM!

File: test/test.js (hunk 1 above)
Comment (joewagner) [reply test/test.js:267]: This repo doesn't have hardhat as a dependency.  Should we add it so we can get the accounts?

**Comment by carsonfarmer** [comment]
Yeh, just add it as a dev dev and that should be fine.

**Comment by joewagner** [comment]
> Yeh, just add it as a devDependency and that should be fine.I wouldn't be surprised if there are other hardhat things we'll need in the future.

I'm running into an issue because this isn't a hardhat project. When I try to `import { ethers } from "hardhat"` it results in `HardhatError: HH1: You are not inside a Hardhat project.`.
What if we cleanup the hardcoded strings and with a module that does a simple export of a wallet for each account?

**Comment by carsonfarmer** [comment]
Naw, doesn't sound worth it. Leave it as is for now. No point in holder up a perfectly good PR with something like that.

**Pull request merged**